CHANGES.rst

```diff
@@ -3,13 +3,22 @@ Version 0.6.0
 
 New features
 ^^^^^^^^^^^^
+- HelioPy has been integrated with SunPy TimeSeries and AstroPy Units. All of
+  HelioPy modules now return units.
+- Added a new :func:`.data.util.cdf_units` function that can extract UNIT attribute from
+  the CDF files.
 
 Backwards incompatible changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - :func:`.data.helios.merged`, :func:`.data.helios.mag_4hz`,
   :func:`.data.helios.corefit` and :func:`.data.helios.mag_ness` no longer take
   a `verbose` keyword argument. :issue:`467`
+- HelioPy no longer returns a Pandas DataFrame.
```

Comment:
Can this be something like "The methods in :mod:`heliopy.data` no longer return...".

(BTW if you want to learn more about sphinx markup then http://www.sphinx-doc.org/en/stable/markup/inline.html is a useful page)

Reply:
Thanks for the link! Sphinx is pretty new to me, but I find it an extremely great way to document stuff!